rainbowstream/py3patch.py

```diff
@@ -8,11 +8,14 @@
     from html.parser import HTMLParser
     from urllib.error import URLError
 
-unescape = HTMLParser().unescape
+# unescape = HTMLParser().unescape
 # According to https://github.com/python/cpython/blob/master/Lib/html/parser.py#L547 ,
 # in python 3.5 maybe I should use
 # from html import unescape
-# but it is a far-future story:)
+# ~~but it is a far-future story:)~~ no longer. 
+# HTMLParser().unescape is deprecated since Python 3.4 and is unsafe.
+# you should use html.unescape instead, like you've said
+from html import unescape
```

Comment:
This line should be included inside the condition that sys.version[0] != 2 at L8~9. Also `unescape = HTMLParser().unescape` need to be restored inside sys.version[0] == "2". 

I am thinking of dropping Python2 support in future release, but as of today this file is made of supporting both Python 2 and 3, then it should work under Python2.

Reply:
I applied these changes in my last commit.